src/Symfony/Component/Validator/Mapping/Cache/CacheInterface.php

```diff
@@ -17,6 +17,8 @@
  * Persists ClassMetadata instances in a cache.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @deprecated in Symfony 4.4, will be removed in 5.0.
```

Comment:
Can we trigger a runtime exception also?

Reply:
`@deprecated since Symfony 4.4`